rust/toolchain.bzl

```diff
@@ -101,6 +106,19 @@ def build_rustdoc_test_command(ctx, toolchain, depinfo, lib_rs):
       depinfo.search_flags +
       depinfo.link_flags)
 
+# @TODO(review) How should this be done? cc_* logic seems much more involved.
```

Comment:
@dslomov Can you help with this portion of the review? Thanks.

Reply:
@mhlopko Can you help take a look at this part of the review?

Reply:
In particular, I'm concerned about handling the placement of .so's emitted by setting crate_type=dylib (which is supported by another PR). 

For depending on cc_* rule dylibs I don't think there is any concern, but the cc_* logic is suspiciously complex.

Reply:
Did your concerns about this get resolved? Either way, we probably don't want to submit this with this TODO and below comment intact.

Reply:
No, but I haven't had issues and neither has anyone else, so that's good enough for now.